.github/workflows/build-and-test.yml

```diff
@@ -71,25 +71,24 @@ jobs:
   setup_test:
     name: Run setup test
     runs-on: ubuntu-latest
-    container: precice/ci-ubuntu-1804
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Install & upgrade pip3
+      - name: Install & upgrade pip3 and install OpenMPI
         run: |
-          apt-get -yy update
-          apt-get install -y python3-pip
-          rm -rf /var/lib/apt/lists/*
+          sudo apt-get -yy update
+          sudo apt-get install -y python3-pip libopenmpi-dev
+          sudo rm -rf /var/lib/apt/lists/*
           pip3 install --upgrade --user pip
-      - name: Checkout precice and make required files discoverable
-        run: | 
-          git clone https://github.com/precice/precice.git precice-core
-          mkdir -p precice
-          cp precice-core/src/precice/SolverInterface.hpp precice/SolverInterface.hpp
       - name: Install dependencies
         run: |
           pip3 install --user toml
           python3 -c 'import toml; c = toml.load("pyproject.toml"); print("\n".join(c["build-system"]["requires"]))' | pip3 install -r /dev/stdin
+      - name: Checkout precice and make required files discoverable 
+        run: |  
+          git clone https://github.com/precice/precice.git precice-core 
+          mkdir -p precice 
+          cp precice-core/src/precice/SolverInterface.hpp precice/SolverInterface.hpp 
```

Comment:
Is there a reason for moving `Checkout precice and make required files discoverable` down? If not, I would keep it where it is for a cleaner diff.

Reply:
Done in 0ef71e3.